src/message_interface/mod.rs

```diff
@@ -90,10 +97,10 @@ pub async fn create_message_handler(options: Option<ManagerOptions>) -> crate::R
     Ok(WalletMessageHandler::with_manager(manager))
 }
 
-pub async fn send_message(handle: &WalletMessageHandler, message: Message) -> Response {
+pub async fn send_message(handle: &WalletMessageHandler, message: Message) -> Option<Response> {
```

Comment:
Just wondering, we seem to end up `expect`ing everywhere so why not the unwrap ?

Reply:
Not in the bindings unfortunately :(